src/test/scala/org/broadinstitute/dsde/rawls/jobexec/MethodConfigResolverSpec.scala

```diff
@@ -117,29 +117,27 @@ class MethodConfigResolverSpec extends WordSpecLike with Matchers with TestDrive
   }
 
 
-  "MethodConfigResolver" should {
-    "resolve method config inputs" in withConfigData {
```

Comment:
This is useful to know -- I've had a tendency to pick a particular test spec style to suit the layout and output that would best make sense for the test. Should we be sticking with one spec exclusively?